package/yast2-add-on.spec

```diff
@@ -41,6 +41,8 @@ Requires:       yast2-country
 Requires:       yast2-installation
 # Packager ProductLicense#HandleLicenseDialogRet allowing "refuse" action
 Requires:       yast2-packager >= 4.2.16
+# "raw_name" in Pkg.SourceEditGet/Set()
```

Comment:
I've checked that this dependency is correct for master (despite the 4.2 minor version)